semantic_release/cli/config.py

```diff
@@ -145,7 +145,7 @@ def resolve_env_vars(cls, val: Any) -> str | None:
     @model_validator(mode="after")
     def set_default_token(self) -> Self:
         # Set the default token name for the given VCS when no user input is given
-        if not self.token and self.type in _known_hvcs:
+        if not self.token and self.type in _known_hvcs:  # noqa: SIM102
```

Comment:
I think this reads cleaner on two lines, so left this and the other vs. resolving, but we can do on a single line if you'd like.

Reply:
Leave it as the single line, because if you make it 2 this entire thing becomes 3 nested ifs which is more hideous.